local-requirements.txt

```diff
@@ -1,5 +1,4 @@
 black==22.6.0
-pytest==7.1.2
```

Comment:
Note: This got removed, so we install the Pytest which the user is installing (and its not pinned anymore), as per setup.py.